db/migrations/20230202110100-add-repo-sync-state-fk.js

```diff
@@ -3,8 +3,9 @@
 module.exports = {
 	up: async (queryInterface) => {
 		//refer to doc https://sequelize.org/v5/class/lib/query-interface.js~queryinterface#instance-method-addConstraint
-		await queryInterface.addConstraint("RepoSyncStates", ["subscriptionId"], {
+		await queryInterface.addConstraint("RepoSyncStates", {
 			type: "foreign key",
+			fields: ["subscriptionId"],
```

Comment:
Outside of all the update to type checking in Sequelize v6, this was the only other change we needed to make.